src/admin/dumb_components/custom_email_form.js

```diff
@@ -1,19 +1,22 @@
 const React = require('react')
-const { map, objOf, zipWith, merge, compose } = require('ramda')
+const { map, objOf, zipWith, merge, compose, indexBy, prop, converge, dissoc } = require('ramda')
 
 export default ({ members, submit }) => {
   const onSubmit = (e) => {
     e.preventDefault()
-    const template = format_message(e.target[0].value)
-    const emailBodies = map(compose(objOf('email_body'), template), members)
-    const emails = zipWith(merge, members, emailBodies)
-    submit(emails)
+    const template = format_message(e.target)
+    const emailBodies = map(compose(objOf('content'), template), members)
+    const emailsArr = zipWith(merge, members, emailBodies)
+    const setEmailKey = map(compose(indexBy, prop), ['primary_email', 'secondary_email'])
```

Comment:
So basically the difference is whether `format_message` gets called in the reducer or in the component? Ideally it would be called in the reducer. It sounds like you foresee a problem with that though?